acm/src/main/java/org/literacybridge/acm/tbbuilder/CreateForV2.java

```diff
@@ -690,6 +691,23 @@ private void addSystemDirFilesToImage() throws IOException {
                     dos.write(publicKey);
                 }
             }
+
+            //
+            Integer volumeStep = ACMConfiguration.getInstance().getCurrentDB().getVolumeStep();
+            if (volumeStep != null) {
+                File targetConfigPath = new File(targetSystemDir, "config.txt");
+                String text = "volume_step:" + volumeStep;
```

Comment:
You declared "  public final static String VOLUME_STEP = "volume_step";", so use it.